operators/syndesis/1.13.0/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM scratch
+
+# Core bundle labels.
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=syndesis
+LABEL operators.operatorframework.io.bundle.channels.v1=1.13.0
```

Comment:
Should the channels not be 1.13.x
See -> https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/operators/syndesis/1.12.0/Dockerfile#L8